.github/workflows/prettier-check.yml

```diff
@@ -0,0 +1,14 @@
+name: Check formatting with Prettier
+
+on: push
```

Comment:
Can we have this run on pull requests? It doesn't make sense to check it after it's merged.